### ISSUE TITLE
Navigation issue, going Back

### ISSUE THREAD

**Issue by mpatton125** [izderadicka/audioserve-android#1]
Hi there,

Just noticed a navigation issue.  Can be recreated as follows:
1) Select "Browse Collections"
2) Select a collection
3) Select an audiobook
4) Press the <- back arrow up top left of the screen
5) Am now back at the "Select Online Collection" screen, and not the list of audiobooks as expected.
Pressing the Android OS back button works as expected, in the same situation.

Obviously only a minor issue.

Regards,

Mike

**Comment by izderadicka** [comment]
Actually this one is intentional, back on top nav bar is always going to root of current navigation ( collections, recently played, bookmarks - whatever you started with).  To go back to previous screen use the android back button on bottom screen).  Initially I have same behavior also for top back arrow that I thought it useless to have same behavior for two buttons - so top is is to jump to navigation root.

**Comment by mpatton125** [comment]
Fair enough - but you have to press it twice to go to the root, not once?
Doesn't feel intuitive.

**Comment by izderadicka** [comment]
Maybe - I checked some Android guidelines in past - they recommend too that top back arrow should behave differently then standard back button,   let's have this as an potential enhancement
